rele/client.py

```diff
@@ -49,8 +49,7 @@ def __init__(self):
 
     def publish(self, topic, data, blocking=False, **attrs):
         payload = JSONRenderer().render(data)
-        logger.info(f'Publishing to {topic}',
-                    extra={'pubsub_publisher_attrs': attrs})
+        self._save_log(attrs, topic)
```

Comment:
A log is not "save", a message is "logged". To be consistent with how the subscriber logging it done, I would instead keep the logging inline and add a `_build_metrics` method for the publisher.

Reply:
Fixed. I add symmetry to publishers and subscribers